Sources/JXKit/JXValue.swift

```diff
@@ -584,26 +584,23 @@ extension JXValue {
 }
 
 extension JXValue {
-
+    
     /// Calls an object as a function.
     ///
     /// - Parameters:
     ///   - arguments: The arguments to pass to the function.
     ///   - this: The object to use as `this`, or `nil` to use the global object as `this`.
     /// - Returns: The object that results from calling object as a function
-    @discardableResult @inlinable public func call(withArguments arguments: [JXValue] = [], this: JXValue? = nil) throws -> JXValue {
```

Comment:
I had to remove @inlinable from some places because I was calling them from non-@inlinable code. I'd like to just remove all the @inlinable stuff because it's so viral and we don't know if we're getting any benefit from it, or even possibly doing harm.

Reply:
I agree – I've never seen much evidence of performance gains from @inlinable in JXKit (although it can be a big boost with other sorts of libraries). Go ahead and nuke any `@inlinable`s you see, but maybe time a few runs of JXKitTests before and after to ensure we don't have any egregious regressions.